plugins/python/test/NumpyReaderTest.cpp

```diff
@@ -236,3 +236,60 @@ TEST(NumpyReaderTest, threedim_col)
     }
 }
 
+
+TEST(NumpyReaderTest, threedim_row_script)
+{
+    StageFactory f;
+
+    Options opts;
+    opts.add("filename", Support::datapath("plang/threedim.py"));
+    opts.add("order", "row");
+    opts.add("function", "load");
+    opts.add("module", "threedim");
+    opts.add("fargs",  Support::datapath("plang/threedim.npy"));
```

Comment:
This seems wrong.  The function takes no args, right?

Reply:
The user function can take in the user's args. For example, in the test, we take in a reference to a npy name to load with our python script and pass back to the `readers.numpy`